db/seeds/permissions.rb

```diff
@@ -101,6 +103,7 @@ def create_permissions(permission_map)
   'article' => %i[read],
   'article_comment' => %i[create read],
   'board_room_presence' => %i[read],
+  'study_room_presence' => %i[read],
```

Comment:
not so sure about this one. why do old members need to know this?

Reply:
it is the same type of information as board room presence, and oudleden are able to view that so that is why i made this choice, however, I do agree and this is not strictly needed.

Reply:
removing the permission for oudleden to view both would be the most logical thing. in my opinion. How do you think about it?